doc/languages.rst

```diff
@@ -949,6 +951,19 @@ to view the docstring of the syntax checker.  Likewise, you may use
          A list of additional library directories. Relative paths are relative
          to the buffer being checked.
 
+   .. syntax-checker:: rust-cargo-clippy
```

Comment:
In terms if errors reported, this checker is a superset of `rust-cargo`.
But in term of options, only the `flycheck-rust-args` and `flycheck-cargo-rustc-args` options are used by this checker.  Most of the options of the others rust checkers doesn't apply to this checker.